bin/alluxio

```diff
@@ -8,9 +8,10 @@ fi
 BIN=$(cd "$( dirname "$0" )"; pwd)
 
 function printUsage {
-  echo "Usage: alluxio COMMAND "
-  echo "where COMMAND is one of:"
-  echo -e "  format [-s]   \t Format Alluxio (if -s specified, only format if underfs is local and doesn't already exist)"
+  echo "Usage: alluxio COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
```

Comment:
the -D options is not meaningful for all commands ... so I think the current usage is misleading ... it might make more sense to group the commands in two groups (one that ends up starting a JVM and another that does not) and use `JAVA_OPTIONS` instead of GENERIC_COMMAND_OPTIONS ... also, it might make sense to do the following:

`alluxio COMMAND [JAVA_OPTIONS --] [COMMAND_ARGS]` so that we can simply look for `--` to decide which arguments are JVM options and which arguments should be passed to the Java program